applications/FluidDynamicsApplication/custom_strategies/strategies/fs_strategy.h

```diff
@@ -642,8 +642,10 @@ class FSStrategy : public SolvingStrategy<TSparseSpace,TDenseSpace,TLinearSolver
 
         double Ratio = NormDv / NormV;
 
-        if ( BaseType::GetEchoLevel() > 0 && rModelPart.GetCommunicator().MyPID() == 0)
-            std::cout << "Fractional velocity relative error: " << Ratio << std::endl;
+        if ( BaseType::GetEchoLevel() > 0 && rModelPart.GetCommunicator().MyPID() == 0){
```

Comment:
Do we need to check for MyPID here? Logger handles it by itself. We can remove it safely from here ryt?

Reply:
perfect ! I will remove it then !